lib/wrappers/transform-node.js

```diff
@@ -71,7 +75,13 @@ module.exports = class TransformNodeWrapper extends NodeWrapper {
         fs.mkdirSync(this.outputPath);
       }
 
-      resolve(this.callbackObject.build());
+      let obj = {};
+      this.inputNodeWrappers.forEach((wrap, idx) => {
+        let revision = this.inputRevisions.get(wrap);
+        obj[idx] = revision ? revision.changed : true;
+      });
+
+      resolve(this.callbackObject.build({ changedNodes: obj }));
```

Comment:
Shouldn't this change also require a corresponding change to the TS interfaces?

Reply:
Yes, it would require a change